apps/projects/api.py

```diff
@@ -15,9 +15,7 @@ class AppProjectsViewSet(viewsets.ReadOnlyModelViewSet):
     lookup_field = "slug"
 
     def get_queryset(self):
-        return Project.objects.filter(
-            is_draft=False, is_archived=False, is_app_accessible=True
-        )
+        return Project.objects.filter(is_draft=False, is_archived=False)
```

Comment:
looking at this, I wonder if this will show private projects? if so we definitely don't want to do that, maybe we can add a test for it to make sure they are not included ? this will also answer the question then

Reply:
on a side note: maybe we can add a fixme/todo in line 12 to rename it from `AppProjectsViewSet` to `ProjectViewSet` as this is no longer app specific

Reply:
And what about past projects? I think @TeaJayyy needs to step in and tell us if the API should only show current and future public projects, so no private or semi-private.